meinberlin/apps/budgeting/assets/disable_contact.js

```diff
@@ -0,0 +1,43 @@
+function disableContact (disable, disableTextfield) {
+  document.getElementById('id_contact_email_0_0').disabled = disable
+  document.getElementById('id_contact_email_0_1').disabled = disable
+  document.getElementById('id_contact_email_1').disabled = disableTextfield
+  document.getElementById('id_contact_phone').disabled = disable
+}
+
+$(document).ready(function () {
```

Comment:
In order to avoid jQuery and to make sure it works in embedded cases:
```
function foo () {
...
}
document.addEventListener('DOMContentLoaded', foo, false)
document.addEventListener('a4.embed.ready', foo, false)
```